### ISSUE TITLE
Reports - max_count

### ISSUE THREAD

**Issue by Ldvt** [python-amazon-mws/python-amazon-mws#80]
Hello,
I have (had) an issue with Reports.get_report_list()

If I call get_report_list(max_count=100), I run into the error 

  File "/home/amzteu/app/bin/mws.py", line 85, in calc_request_description
    encoded_value = quote(params[key], safe='-_.~')
  File "/usr/lib64/python2.7/urllib.py", line 1280, in quote
    if not s.rstrip(safe):
**AttributeError: 'int' object has no attribute 'rstrip'**

After some debug, this can be easily fixed by calling 
`get_report_list(max_count='100')`
Basically passing a text instead of an integer (line 438, param max_count)

In my opinion max_count makes more sense as a number, so it would be better to edit line 444 into `MaxCount=str(max_count))`

Thanks,
Alberto

**Comment by elcolumbio** [comment]
Hey Alberto,
good catch. Every parameter is in the end converted to a string.

In the develop branch all those things are fixed with the clean_params method from #65 .

We soon release the 1.00 there you will get that experience. Or use the develop version ;)

**Comment by Ldvt** [comment]
Hello Florian,
Thanks for your message, I have fixed my code.
In the meanwhie, I needed update_report_acknowledgements (still unavailable
in develop branch
https://github.com/python-amazon-mws/python-amazon-mws/blob/develop/mws/apis/reports.py
, lines 218..220), so I wrote it

I have tested it just few times and it works fine.

Quite unfamiliar to push it into github.
Hope this helps.


def update_report_acknowledgements(self, report_ids=()):
data = dict(Action='UpdateReportAcknowledgements')
data.update(utils.enumerate_param('ReportIdList.Id.', report_ids))
return self.make_request(data)

**Comment by elcolumbio** [comment]
I am new too. I use a gui gitkraken. But in this case easiest is to use the gui from github.
1. Fork the project.
2. Switch to develop branch
3. edit the code and commit to your fork
4. make a pull request: your devevelop branch to the develop branch here
I am not 100% sure that are the correct steps, but i think its like that.

If we you are not using the github gui you need to understand more concepts, like to add remotes.
